week04/code.js

```diff
@@ -0,0 +1,83 @@
+const btns = document.getElementById("btns");
+const start = document.getElementById("start");
+const reset = document.getElementById("reset");
+
+const human = document.getElementById("Hchoice");
+const com = document.getElementById("Cchoice");
+
+const score = document.getElementById("score");
+const result = document.getElementById("result");
+
+const elements = ["✌", "✊", "✋"];
+
+let myChoice = null;
+let Cchoice = null;
+
+let myScore = 0;
+let comScore = 0;
+
+const clickStart = () => {
+  document.querySelectorAll("#btns button").forEach((btn) => {
+    btn.removeAttribute("disabled");
+  });
+  myChoice = null;
+  Cchoice = Math.floor(Math.random() * 3);
+
```

Comment:
함수가 여러 번 실행되었을 때 전역 변수로 선언한 timer가 중단되지 않고 반복적으로 실행되므로, 클릭했을 때 아직 이전 interval이 실행 중이었다면 중단하고 재실행될 수 있도록 interval 시작 전에 clearInterval을 해줍니다.

```suggestion
clearInterval(timer);
```